python/test/unit/fem/test_fem_pipeline.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (C) 2019 Jorgen Dokken
+#
+# This file is part of DOLFIN (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+
+import numpy as np
+import pytest
+import sympy as sp
+
+from dolfin import (MPI, DirichletBC, Function, FunctionSpace, TestFunction,
+                    TrialFunction, UnitCubeMesh, UnitIntervalMesh,
+                    UnitSquareMesh, solve)
+from dolfin.cpp.mesh import CellType
+from ufl import SpatialCoordinate, dx, grad, inner
+
+
+class ManufacturedSolutionPoisson():
```

Comment:
Seems a bit heavy-weight to have a class for this. Could a single function suffice?